src/System.Security.Cryptography.OpenSsl/dir.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.0.2</AssemblyVersion>
```

Comment:
Assembly version should still be 4 parts.

Reply:
Oops my bad. Didn't mean to do that.